runtime/queries/javascript/textobjects.scm

```diff
@@ -0,0 +1,35 @@
+(function_declaration
+  body: (statement_block)) @function.around
+
+(function
+  body: (statement_block)) @function.around
+
+(export_statement
+  (function_declaration) @function.around) @function.around.start
+
+(arrow_function
+  body: (statement_block)) @function.around
+
+(method_definition
+  body: (statement_block)) @function.around
+
+(class_declaration
+  body: (class_body) @class.inside) @class.around
+
+(export_statement
+  (class_declaration) @class.around) @class.around.start
+
+(export_statement
+  (class) @class.around) @class.around.start
```

Comment:
what are the `.start` captures here? I think you should be able to remove them

Reply:
thanks , i  fix it